test/verify/check-superuser

```diff
@@ -382,7 +382,7 @@ class TestSuperuserDashboard(MachineCase):
         b.wait_in_text(".super-channel span", 'access-denied')
 
         # Make sure sudo asks for a password
-        self.machines["machine2"].execute("rm /run/sudo/ts/admin")
+        self.machines["machine2"].execute("rm -f /run/sudo/ts/admin")
```

Comment:
Hmm, did the path just move somewhere else? Sudo needs to remember this *somewhere*.

Reply:
After discussion with @mvollmer:
> `<mvollmer>` yeah, I think we can just remove that line for every OS.

Lets do that it in a different PR though.

Reply:
I would say that we do not need to remove this file for any OS, since Cockpit runs `sudo -k`. The test might have been written before Cockpit did that, and it might thus be a left over.
However, the file _is_ there, so maybe I am confused.
A more portable way to kill the cached credentials would be `m.execute("su admin sudo -k")` I guess.

Reply:
https://github.com/cockpit-project/cockpit/pull/14521